components/datetime/src/format/zoned_datetime.rs

```diff
@@ -13,15 +13,15 @@ use writeable::Writeable;
 use super::datetime;
 use super::time_zone;
 
-pub struct FormattedZonedDateTime<'l, T>
+pub struct FormattedZonedDateTime<'l, 'd, T>
 where
     T: ZonedDateTimeInput,
 {
-    pub(crate) zoned_datetime_format: &'l ZonedDateTimeFormat<'l>,
+    pub(crate) zoned_datetime_format: &'l ZonedDateTimeFormat<'d>,
```

Comment:
@nordzilla I had to add a second lifetime parameter to `FormattedZonedDateTime` because the lifetime on the `ZonedDateTimeFormat` can be different than the lifetime of the borrow.  I ran into this when refactoring the DateTime test.